apps/web/lib/linkValueToString.ts

```diff
@@ -0,0 +1,41 @@
+import { TFunction } from "next-i18next";
+
+import { LocationType } from "./location";
+
+/**
+ * Use this function to translate booking location value to a readable string
+ * @param linkValue
+ * @param translationFunction
+ * @returns
+ */
+export const linkValueToString = (
+  linkValue: string | undefined | null,
+  translationFunction: TFunction
+): string => {
+  const t = translationFunction;
+  if (!linkValue) {
+    return translationFunction("no_location");
+  }
+  switch (linkValue) {
+    case LocationType.InPerson:
+      return t("in_person_meeting");
+    case LocationType.UserPhone:
+      return t("user_phone");
+    case LocationType.GoogleMeet:
+      return t("cal_provide_google_meet_location");
+    case LocationType.Zoom:
+      return t("cal_provide_zoom_meeting_url");
```

Comment:
'Cal will provide a video meeting URL.' doesn't really fit for current location as the meeting Url is already provided at that time. Should we say something like 'Meeting url in confirmation email'. Even though, confirmation email is also not perfectly correct because when the location was already changed before the current meeting URL is in the email from the location change not the actual confirmation mail. @alannnc @Jaibles what do you think?